### PR TITLE
218/jake/review screens

### PR DESCRIPTION
**Pull request by jakegraves-figure**

### CODE REVIEW

File: lib/common/widgets/pw_slider.dart [hunk 1]
```diff
@@ -35,35 +35,147 @@ class _PwSliderState extends State<PwSlider> {
...
+                    activeTickMarkColor: Colors.transparent,
+                    inactiveTickMarkColor: Colors.transparent,
+                    valueIndicatorColor: theme.colorScheme.secondary350,
+                    valueIndicatorTextStyle: theme.textTheme.footnote
+                        .copyWith(color: theme.colorScheme.neutral800),
+                  ),
+                  child: Slider(
+                    min: widget.min,
+                    max: widget.max,
+                    label: _value == 1.25
```
Comment (royd) [inline lib/common/widgets/pw_slider.dart:86]: Is there already a `const` somewhere for `1.25` that could be used?

File: lib/screens/home/staking/staking_confirm/staking_confirm_base.dart [hunk 2]
```diff
@@ -36,49 +36,43 @@ class _StakingConfirmBaseState extends State<StakingConfirmBase> {
...
+          padding: EdgeInsets.symmetric(horizontal: Spacing.large),
           children: [
             ...widget.children,
-            PwListDivider(
-              indent: Spacing.largeX3,
-            ),
+            PwListDivider.alternate(),
             PwSlider(
               title: Strings.stakingConfirmGasAdjustment,
               startingValue: 1.25,
```
Comment (royd) [inline lib/screens/home/staking/staking_confirm/staking_confirm_base.dart:78]: Could this be a unified `const`?

File: lib/screens/home/staking/staking_delegation/staking_delegation_bloc.dart [hunk 3]
```diff
@@ -194,6 +201,10 @@ class StakingDelegationDetails {
 
     return hashDelegated > remainingHash;
   }
+
+  String get hashFormatted {
+    return "$hashDelegated HASH";
```
Comment (royd) [inline lib/screens/home/staking/staking_delegation/staking_delegation_bloc.dart:206]: Might be good to get `HASH` from a central location.

File: lib/screens/home/staking/staking_delegation/staking_undelegation_screen.dart [hunk 4]
```diff
@@ -48,6 +48,7 @@ class _StakingUndelegationScreenState extends State<StakingUndelegationScreen> {
       "",
       SelectedDelegationType.undelegate,
       widget.account,
+      null,
```
Comment (royd) [inline lib/screens/home/staking/staking_delegation/staking_undelegation_screen.dart:51]: Named parameters could improve readability

File: lib/screens/home/staking/staking_screen_bloc.dart [hunk 5]
```diff
@@ -57,7 +57,9 @@ class StakingScreenBloc extends PwPagingCache {
       );
 
       final rewards = await _validatorService.getRewards(
-          _account.publicKey!.coin, _account.publicKey!.address);
+        _account.publicKey!.coin,
```
Comment (royd) [inline lib/screens/home/staking/staking_screen_bloc.dart:60]: Thinking about bringing `TransactableAccount` with a non-null `publicKey` back...

File: lib/services/models/commission.dart [hunk 6]
```diff
@@ -63,8 +64,8 @@ class Commission {
   }
 
   String get formattedBondedTokens {
-    var tokens = bondedTokensCount.nhashToHash(fractionDigits: 7).split('.');
-    return '${tokens[0].formatNumber()}.${tokens[1]} HASH';
+    var tokens = bondedTokensCount.nhashToHash(fractionDigits: 7);
```
Comment (royd) [inline lib/services/models/commission.dart:67]: I think `denom_util` should be used and `int_extension` removed. Seems like the `fractionalDigits` should be unified everywhere.

**Review by royd — CHANGES_REQUESTED** [review]

File: lib/common/widgets/pw_slider.dart (hunk 1 above)
Comment (jakegraves-figure) [reply lib/common/widgets/pw_slider.dart:86]: Yes!

File: lib/screens/home/staking/staking_confirm/staking_confirm_base.dart (hunk 2 above)
Comment (jakegraves-figure) [reply lib/screens/home/staking/staking_confirm/staking_confirm_base.dart:78]: It will be eventually.

**Review by royd — APPROVED** [review]

**Pull request merged**